### PR TITLE
Simplify MQTT configuration and some cleanup

### PR DESCRIPTION
**Pull request by tomrosenback**
Remove unecessary logging
Use MQTT service configuration if available
Update docs and changelog
Update version

Solves #9

### CODE REVIEW

File: eda-modbus-bridge/CHANGELOG.md [hunk 1]
```diff
@@ -6,12 +6,19 @@ application itself.
...
 
 ## 1.0.1
 
 - Update to [eda-modbus-bridge 2.0.0](https://github.com/Jalle19/eda-modbus-bridge/releases/tag/2.0.0)
 - Require MQTT service
 - Delay startup
+
+## 1.0.0
+
+Initial release as a proper Home Assistant add-on!
```
Comment (Jalle19) [inline eda-modbus-bridge/CHANGELOG.md:24]: I see that you changed the order, thanks :+1:

**Review by Jalle19 — APPROVED** [review]
Nice addition, LGTM!

**Comment by Jalle19** [comment]
I tried this but all I got was "MQTT not found, using addon configuration" or whatever the message was :shrug: Did it work for you?

**Comment by tomrosenback** [comment]
Sure works, for me it says 
[15:52:40] INFO: MQTT service found, using service config.

You have MQTT installed in HA? core_mosquitto addon.

**Comment by tomrosenback** [comment]
You have it configured in configuration.yaml also?

**Comment by Jalle19** [comment]
No, Mosquitto is running elsewhere. I just have the MQTT integration configured.

**Comment by tomrosenback** [comment]
Ok, well that explains it, and then it works as it should.

**Pull request merged**